packages/cli/src/commands/new.ts

```diff
@@ -14,7 +14,6 @@ import {runPrisma} from "./prisma"
 
 export interface Flags {
   js: boolean
-  "skip-install": boolean
```

Comment:
I removed this and related code, as this is dead code that now we do update before asking for dependency installation.

Reply:
Aren't those two separate things? If you pass `skip-install` it should assume automatically that you don't want to install dependencies and don't show the `install deps?` prompt.

Reply:
I think this line: https://github.com/blitz-js/blitz/pull/2791/files#diff-6290fc15ecf346e1efd5672845e50c464b898dd25364403fe918314da4d838ffR95 should go after `determinePkgManagerToInstallDeps`

Reply:
Yes I totally agree about the `skip-install` part, however this was mainly used when the upgrade Blitz process finished to rerun the whole `new` command without the `install deps?` prompt. As now the upgrade process is before we don't need to restart the `new` command without this prompt. Also the `skip-install` was hidden, so not used by user.

Reply:
Indeed, I'll swipe these lines.